Add/roadmap (#22)
WordPress/ai: ROADMAP.md

# AI Experiments Plugin Roadmap

This roadmap outlines the planned development of the AI Experiments plugin from its early prototypes through the [WordPress 6.9](https://make.wordpress.org/core/6-9/) release and into 2026.  Each milestone includes goals, feature targets, and integration points with key components across the WordPress AI ecosystem.

## Milestone 0.1.0 - Plugin Foundations and First Feature Integration

**Target:** Early October 2025

### Goals

- [ ] Scaffold the initial plugin structure in the `WordPress/ai` GitHub repository
- [ ] Create a basic admin settings screen with a toggle for enabling experimental features

> - [ ] Integrate the PHP AI Client SDK (or alternatively WP AI Client SDK) with a simple service provider setup

Reviewer: This should really be the latter. I realize the WP AI Client SDK hasn't launched yet, so we may need to speed up things there, but directly using the PHP AI Client SDK instead should only be done as a temporary workaround, if absolutely necessary.
```suggestion
- [ ] Integrate the WP AI Client SDK (and underlying PHP AI Client SDK) with a simple service provider setup
```
